include/crocoddyl/core/actuation-base.hpp

```diff
@@ -52,6 +52,8 @@ class ActuationModelAbstractTpl {
   const std::size_t& get_nu() const { return nu_; };
   const boost::shared_ptr<StateAbstract>& get_state() const { return state_; };
 
+  virtual void get_actuated(const boost::shared_ptr<ActuationDataAbstract>& data, Eigen::Ref<VectorXs>& u) = 0;
```

Comment:
I don't see the objective of this function, could you explain me?

Reply:
It is the inverse of the calc, in order to obtain the actuated control given the final actuation tau.